.github/workflows/run-test-suite.yml

```diff
@@ -0,0 +1,89 @@
+name: run-test-suite
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    services:
+      postgres:
+        image: postgres:10.8
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: password
+        ports:
+          - 3306
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=3
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
```

Comment:
we had some comments in the old travis file here. could you put some of those into here?

Reply:
the only comment that was around this part was that the `ruby/setup-ruby@v1` will automatically get bug fixes because this action is provided by the ruby org on GitHub. it adds any supported Ruby version to PATH for each job run in a workflow. does that need a comment? 

or was there a different comment that i'm forgetting about?